extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -18,6 +18,8 @@ const BackbeatClient = require('../../../lib/clients/BackbeatClient');
 const QueueEntry = require('../utils/QueueEntry');
 const CredentialsManager = require('../../../credentials/CredentialsManager');
 
+const LIMIT = 10;
```

Comment:
It could be interesting to have this configurable. I heard that some of our customers will probably need MPU for performance, and the parallelism factor will be an important variable to control in order to optimize performance for big objects.